lib/fpm/package/deb.rb

```diff
@@ -724,9 +738,20 @@ def fix_provides(provides)
                    "debs don't like underscores")
       provides = provides.gsub("_", "-")
     end
+    
     return provides.rstrip
   end
 
+  def validate_provides()
+    self.provides.each do |provides|
```

Comment:
Similar comment to above with 'requires' -- thoughts on lowercasing instead of deleting?

Reply:
See comment below.